frontend/src/BenchmarkTest.res

```diff
@@ -7,10 +7,10 @@ type testMetrics = {
   metrics: Belt.Map.String.t<float>,
 }
 
-let commitUrl = commit => `https://github.com/mirage/index/commit/${commit}`
-let goToCommitLink = commit => {
+let commitUrl = (repo, commit) => `https://github.com/${repo}/commit/${commit}`
```

Comment:
Similar to my comment for the sidebar, can we use `repo_id` instead of `repo`. My suggestion is to always use `repo_id` to represent `repo_org/repo_name`.

Reply:
Makes sense. On a side note, we're not consistent with our case style. Some variables are `snake_case` (mainly fields from GraphqQL) and the others `camelCase`

Reply:
Yes, I agree those should be consistent. It's all my fault, I've been switching a lot between OCaml and ReScript projects and have a tendency to use underscores without realising it. Please feel free to point this out in reviews.